licensedb/internal/processors/preprocessors.go

```diff
@@ -0,0 +1,23 @@
+package processors
+
+import (
+	"bytes"
+
+	"github.com/hhatto/gorst"
+	"gopkg.in/russross/blackfriday.v2"
+)
+
+// Markdown converts Markdown to plain text. It tries to revert all the decorations.
+func Markdown(text []byte) []byte {
```

Comment:
Too generic and not a verb... `RenderMarkdown`? `MarkdownToPlainText`?

Reply:
maybe we need to find a better name for the package then
currently you'd call `processor.Markdown` maybe it should be called `extract.FromMarkdown`?